common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -59,12 +60,14 @@ private LeaderInquireClient(String zookeeperAddress, String leaderPath) {
         CuratorFrameworkFactory.newClient(mZookeeperAddress, new ExponentialBackoffRetry(
             Constants.SECOND_MS, 3));
     mCLient.start();
+
+    mMaxTry = new TachyonConf().getInt(Constants.ZOOKEEPER_LEADER_INQUIRY_RETRY_COUNT);
```

Comment:
ClientContext can't import into LeaderInquireClient.java, because ClientContext belong to the module "client", and LeaderInquireClient belongs to the module "common".
I think a better way is introducing the TachyonConf by adding a parameter TachyonConf for method "getClient".  @apc999